astropy/units/function/core.py

```diff
@@ -230,6 +230,9 @@ def is_equivalent(self, other, equivalencies=[]):
 
         return self.physical_unit.is_equivalent(other_physical_unit, equivalencies)
 
+    # def get_converter(self, other, equivalencies=[]):
+    #     pass
+
```

Comment:
Currently `FunctionUnitBase` does not expose the `get_converter` method, and I guess they should for consistency. `u.Jy.get_converter(u.ABmag)` works but ` u.ABmag.get_converter(u.Jy)` doesn't.

If we do end up going forward with this PR should I add the functionality in this PR or create a new one?

Reply:
Good point. Essentially, the logic is all in `to()`, so that would need to be changed. My sense is that you have a nice & easy PR here, so do this as follow-up!

Reply:
Perfect! I will create a new PR for that change.